scripts/release_scripts/deploy_test.py

```diff
@@ -204,6 +204,18 @@ def test_invalid_release_version(self):
                 'Current release version has \'.\' character.'):
                 deploy.execute_deployment()
 
+    def test_invalid_release_version_length(self):
+        args_swap = self.swap(
+            sys, 'argv', [
+                'deploy.py', '--app_name=oppiatestserver',
+                '--version=release-1.2.3-invalid-custom'])
```

Comment:
Maybe release-1.2.3-invalid-too-long, just to make it super obvious to readers what the issue is.

Reply:
Done.